D_controllers.md

```diff
@@ -76,21 +87,27 @@ The data world is your oyster, but we won't be covering these options in these g
 
 There are times when we need to communicate with users during the course of an action. Maybe there was an error updating a model. Maybe we just want to welcome them back to the application. For this, we have flash messages.
 
-The `Phoenix.Controller` module provides the `put_flash/3` and `get_flash/2` functions to help us set and retrieve flash messages as a key value pair. Let's set two flash messages in our `HelloPhoenix.PageController` to try this out.
+The `Phoenix.Controller` module provides the `put_flash/3` and `get_flash/2` functions to help us set and retrieve flash messages as a key value pair. Let's set two flash messages in our `HelloPhoenix.PageController` in `web/controllers/page_controller.ex` to try this out.
```

Comment:
I don't think we need to tell the users again where the controller is located, so we can remove, "in `web/controllers/page_controller.ex`".